chrome/content/zotero/xpcom/itemTreeView.js

```diff
@@ -260,6 +261,44 @@ Zotero.ItemTreeView.prototype.setTree = Zotero.Promise.coroutine(function* (tree
 });
 
 
+Zotero.ItemTreeView.prototype.setSortColumn = function() {
+	var colId, col, dir, currentCol, currentDir;
```

Comment:
>I don't get it why they didn't use the chance with ES6 to remove declarators completely

Because that's not backwards compatible.

>// Did I use 'sql' above? Do I need to go check?

(1) Huge functions like that probably need some refactoring. If you can't keep track of what the whole function is doing, it's doing too much. (2) Yes, you should check. You should know what your code is doing in the context of the whole function. (3) A thrown exception for reusing `let foo;` is a whole lot easier to catch than unintentionally overwriting a variable. Strict mode does the same, but I don't think we have that declared in all scripts either.